native/wasmex/src/instance.rs

```diff
@@ -22,14 +24,48 @@ pub struct InstanceResource {
 //
 // * bytes (binary): the bytes of the WASM module
 // * imports (map): a map defining eventual instance imports, may be empty if there are none.
-//   structure: %{namespace_name: %{import_name: {TODO: signature}}}
+//   structure: %{namespace_name: %{import_name: {:fn, param_types, result_types, captured_function}}}
```

Comment:
Oh OK, ✅ marked as done 😎

Reply:
I didn't check this against reality yet (wrote it from memory when I drafted the PR).

So making sure this is actually correct is still a Todo :)